lib/animina_web/components/bookmark/bookmark_component.ex

```diff
@@ -55,6 +55,17 @@ defmodule AniminaWeb.BookmarkComponent do
     end
   end
 
+  # we display the state as an atom in the photo struct, but we need to make sure it is an atom
```

Comment:
When I read this I ask myself WHY do we need to make sure if we already save it as an atom? Please add that this is probably a bug in Ash. Bonus points for adding a link to the bug report you created for Ash.